src/applications/ezr/utils/helpers/form-config.js

```diff
@@ -47,6 +48,41 @@ export function includeTeraInformation(formData) {
   return formData.hasTeraResponse;
 }
 
+export function veteranBornBetween(formData, date1, date2) {
+  return isWithinInterval(new Date(formData?.veteranDateOfBirth), {
+    start: new Date(date1),
+    end: new Date(date2),
+  });
+}
+
+export function canVeteranProvideAgentOrangeResponse(formData) {
+  return (
+    includeTeraInformation(formData) &&
+    veteranBornBetween(formData, new Date(1900, 1, 1), new Date(1965, 12, 31))
```

Comment:
I don't know that we need to do anything here, but this date (`new Date(1900, 1, 1)`) seems like a really common date we use to validate for things, and I wonder if it would make sense to pull it out to a shared const or something at some point? Not like it's really going to change, though 🤷 , but it could increase readability.

Reply:
I agree. I've noticed it's built into some of the platform validation as well. Maybe we can talk about it during our next dev sync?